test/format/ops.c

```diff
@@ -622,6 +688,20 @@ table_op(TINFO *tinfo, bool intxn, iso_level_t iso_level, thread_op op)
     /* Acquire a cursor. */
     tinfo->cursor = table_cursor(tinfo, table->id);
 
+    /*
+     * Predictable replay has some restrictions. Someday we may be able to resolve some of these
+     * restrictions, this may require adding complexity.
```

Comment:
Is this feasible enough to ticket as potential follow-on work. Or does it seem too intractable?

Reply:
This one is tougher.  When we insert a record to a column store, we expand the key space by one, and it's hard to expose that immediately to all threads in a predictable way.  We could "hide" the real end point of the column store for, say 1000 timestamps.  But we have to handle the case when the creator is behind the front runner by > 1000 timestamps.  I think a solution is to only allow the creator of a new key to have a matched lane for the new key.  I'm convincing myself that it could be possible, I'll write up a ticket.

Reply:
WT-10572.  Feels possible, but maybe too complex to pursue unless we're really motivated to more fully test CS with predictable replay.